spec/spec_helper.rb

```diff
@@ -1 +1,4 @@
 require 'nexpose'
+
+def testCamelCaseComment(from: 'hound')
```

Comment:
Unused method argument - `from`. You can also write as `testCamelCaseComment(*)` if you want the method to accept any arguments but don't care about them.<br>Use snake_case for methods.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.